pkg/inject/report.go

```diff
@@ -142,6 +150,24 @@ func checkUDPPorts(t *v1.PodSpec) bool {
 	return false
 }
 
+func checkNodeOS(t *v1.PodSpec) bool {
```

Comment:
Can we rename this to something like `IsWindowsNode()`, as we are returning a bool, but not the name?
Same goes for field in `report`